docs/analysis-workflows/harmonization-dna-pipeline.md

```diff
@@ -0,0 +1,37 @@
+---
+id: dna-pipeline
+title: DNA-Seq Analysis Pipeline
+---
+
+The DNA-Seq analysis pipeline identifies various somatic variant types within Whole Exome Sequencing (WXS) and Whole Genome Squencing (WGS) data. The pipeline starts with a reference alignment step followed by co-cleaning to increase the alignment quality. DNA-Seq analysis is implemented across two main procedures:
+
+- Sequence Alignment
+- Variant Calling
+
+In the future, these procedures will be extended to include:
+
+- Variant Masking
+- Variant Annotation
+- Concencus Calling
+
+## Alignment
+
+Before genomes can be compared for variant analysis, they must be aligned to to a reference genome. In addition to alignment, the alignemtn step includes some data cleanup operations to correct for technical biases and make the data suitable for analysis.

```

Comment:
remove second "to' in "they must be aligned to to"
typo `alignemtn`
but you could also change this to: "In addition to alignment, `this step` includes some data cleanup `that` corrects for technical biases and `makes` the data suitable for analysis."

Reply:
- like the suggestion, but not to remove the word operations.

Reply:
oops, didn't mean to delete that